### ISSUE TITLE
Stripe Payment Plugin Missing Error on Checkout Complete

### ISSUE THREAD

**Issue by adrienshen** [mirumee/saleor#5849]
### What I'm trying to achieve
I'm trying to complete checkout using Stripe

### Steps to reproduce the problem
1. On version 2.10, buy some items, enter in an address and contact details, shipping methods
2. Enter Stripe SK and PK in plugin's configuration
2. Complete checkout with Stripe

### What I expected to happen
Checkout completes

### Screenshots
<img width="1034" alt="Screenshot 2020-07-04 21 09 03" src="https://user-images.githubusercontent.com/12431027/86525337-d0484800-be3a-11ea-8083-fa017f364499.png">

```
ERROR saleor.payment.gateway Error encountered while executing payment gateway. [PID:36919:Thread-3]
Traceback (most recent call last):
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/payment/gateway.py", line 212, in _fetch_gateway_response
    response = fn(*args, **kwargs)
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/plugins/manager.py", line 286, in process_payment
    return self.__run_payment_method(gateway, method_name, payment_information)
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/plugins/manager.py", line 361, in __run_payment_method
    f"Payment plugin {gateway} for {method_name}"
Exception: Payment plugin Stripe for process_payment payment method is inaccessible!
ERROR saleor.payment.gateway Error encountered while executing payment gateway. [PID:36919:Thread-3]
Traceback (most recent call last):
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/payment/gateway.py", line 212, in _fetch_gateway_response
    response = fn(*args, **kwargs)
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/plugins/manager.py", line 286, in process_payment
    return self.__run_payment_method(gateway, method_name, payment_information)
  File "/Users/adrienshen/Documents/Fl/cabinet1/code/sp2/SaleorCabinetAPI/saleor/plugins/manager.py", line 361, in __run_payment_method
    f"Payment plugin {gateway} for {method_name}"
Exception: Payment plugin Stripe for process_payment payment method is inaccessible!
```

It seems like the problem is here:

```
    def get_plugin(self, plugin_id: str) -> Optional["BasePlugin"]:
        # if plugin_id == 'Stripe':
        #     plugin_id = 'mirumee.payments.stripe'
        print('GOT HERE !!! plugin is {}'.format(plugin_id))
        for plugin in self.plugins:
            if plugin.PLUGIN_ID == plugin_id:
                return plugin
        return None

    def fetch_taxes_data(self) -> bool:
        default_value = False
        return self.__run_method_on_plugins("fetch_taxes_data", default_value)
```

plugin_id is 'Stripe', but it's expecting `mirumee.payments.stripe`

I'm not sure where to fix this, or why this is happening.

**System information**
Operating system: macOS 10.15.4 (19E287)
Chrome Browser

**Comment by adrienshen** [comment]
I could hack it together like in the code sample above and it'll work, but not sure if that's the best solution.

**Comment by maarcingebala** [comment]
Hey @adrienshen, we were trying to reproduce this error in Saleor 2.10.1 but it works fine. Does it happen on your local environment or some production? I think it may be related to some old data in your database. We used to store the name of the payment gateway used in checkout/order e.g. "Stripe" but we changed it to ID in form of "mirumee.payments.stripe". There was also a migration in payments module to update all existing payment objects. It seems like in your case some old payment instance was still present in the database.
